observability-lib/core-node/component.go

```diff
@@ -433,7 +448,7 @@ func logPoller(p Props) []cog.Builder[dashboard.Panel] {
 		"",
 		common.LegendPlacementBottom,
 		utils.PrometheusQuery{
-			Query:  `avg by (evmChainID) (rate(log_poller_logs_inserted{job=~"$instance", evmChainID=~"$evmChainID"}[$__rate_interval]))`,
+			Query:  `avg by (evmChainID) (rate(log_poller_logs_inserted{` + p.PlatformOpts.LabelQuery + `}[$__rate_interval]))`,
```

Comment:
@Atrax1 for my understanding: how does `evmChainId` end up in p.PlatformOpts.LabelQuery now?

Reply:
it does not but we are averaging by evmChainId so it shows value for each evmChainId and then we can use legend to filter

Reply:
value of p.PlatformOpts.LabelQuery is depending on https://github.com/smartcontractkit/chainlink-common/blob/main/observability-lib/core-node/platform.go#L41 and variables at the top of dashboard where we filter on is defined here : https://github.com/smartcontractkit/chainlink-common/pull/566/files#diff-00b43326a56b37178a046bdc93b3f19a6484111b97fbf9c8af147c9aa2e50076R86

Reply:
My misunderstanding was that we still wanted to filter by evmChainID. We do not, you want a view that has all the chains together.